manager/integration/tests/test_recurring_job.py

```diff
@@ -241,15 +241,16 @@ def test_recurring_job(set_random_backupstore, client, volume_name):  # NOQA
                                          groups=[DEFAULT])
 
     wait_until_begin_of_an_even_minute()
-    # wait until the 20th second of an even minute
+    # wait until the 30th second of an even minute
     # make sure that snapshot job happens before the backup job
-    time.sleep(20)
+    time.sleep(30)
```

Comment:
Will adjusting this sleep time affect the pass rate of this test case?

Reply:
Yes, I have executed the private image test 10 times, and the result passed on v1.6.3-rc1:
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/7587/testReport/tests/test_recurring_job/. 
![Screenshot_20240912_155328](https://github.com/user-attachments/assets/96ce0f57-b04b-42a6-954c-518535ca52dd)


However, I must admit that, at the moment, I don't have any clear ideas about what increasing the time would actually improve.

Reply:
That’s fine. It’s possible that the process occasionally takes longer than 20 seconds to initiate.